### ISSUE TITLE
Crash Daemon

### ISSUE THREAD

**Issue by xakep-bs** [Lessica/TrollSpeed#39]
If add subview to hudrootviewconntroller. In subview scrollview when swiping must crash daemon.

**Comment by Lessica** [comment]
So what?

**Comment by xakep-bs** [comment]
I just wanted to add an auxiliary external menu to your source code, through which settings can be configured. But when I added it, the ScrollView started crashing when swiping. It might be due to gesture conflicts, but I'm not entirely sure which ones since I removed all the gestures, but the problem persisted. There are a lot of messages in the logs; I can send them to you if you're interested.Also, when I added a new tap gesture to the RootHUDViewController (i.e., the menu is supposed to appear after a triple tap on the screen), it didn’t work either.


[23082024_113310_894_iPhone_(Empty).txt](https://github.com/user-attachments/files/16735936/23082024_113310_894_iPhone_.Empty.txt)
